LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018 Mauricio Cousillas <mauricio.cousillas@moove-it.com>
```

Comment:
maybe we should change it to moove-it like here https://github.com/moove-it/Swift-Project-Template/blob/master/LICENCE